### PR TITLE
Bump dependency versions and fix deprecated API calls

### PR DESCRIPTION
**Pull request by urieli**

### CODE REVIEW

**Comment by urieli** [comment]
This is a very minor update, but since it's my first time on this project, @satabin can you take a quick look?

File: build.sbt [hunk 1]
```diff
@@ -53,12 +53,12 @@ lazy val root = project.in(file("."))
     resolvers += "Sonatype Snapshots" at "https://oss.sonatype.org/content/repositories/snapshots/",
     organization := "com.safety-data",
     name := "akka-persistence-redis",
-    version := "0.4.0",
+    version := "0.5.0-SNAPSHOT",
```
Comment (satabin) [inline build.sbt]: For such a small change, I think a patch release is better. That would then be `0.4.1`.
If tests are all passing, then you can directly publish the final patch release.

File: src/test/scala/akka/persistence/query/journal/redis/EventsByPersistenceIdSpec.scala [hunk 2]
```diff
@@ -67,7 +67,7 @@ class EventsByPersistenceIdSpec extends AkkaSpec(EventsByPersistenceIdSpec.confi
       src.map(_.event).runWith(TestSink.probe[Any])
         .request(2)
         .expectNext("a-1", "a-2")
-        .expectNoMsg(500.millis)
+        .expectNoMessage(500.millis)
```
Comment (satabin) [inline src/test/scala/akka/persistence/query/journal/redis/EventsByPersistenceIdSpec.scala:70]: Grumpf, changing names in a patch version (I mean in akka, not here) is so bad...

**Review by satabin — CHANGES_REQUESTED** [review]

File: src/test/scala/akka/persistence/query/journal/redis/EventsByPersistenceIdSpec.scala (hunk 2 above)
Comment (urieli) [reply src/test/scala/akka/persistence/query/journal/redis/EventsByPersistenceIdSpec.scala:70]: They didn't change completely, only deprecated the old name :) Still, I don't like to see warnings when I compile!

Comment (satabin) [reply src/test/scala/akka/persistence/query/journal/redis/EventsByPersistenceIdSpec.scala:70]: Then, LGTM :)

**Pull request merged**